Automattic/simplenote-android
# Add Styled Widgets (Part 2)

### Fix
This is the second part in a series of pull requests to add styled widgets to the app.  The pull requests are split into parts in an attempt to make the changes smaller.  This part adds the selected app style to the note list widget.  See the screenshots below for illustration.

![add_styled_widgets_2](https://user-images.githubusercontent.com/3827611/97364428-ae4f5700-1869-11eb-879d-d498ef5be107.png)

### Test
The first steps add the **_Note List (Light)_** and **_Note List (Dark)_** widgets to the home screen.  These steps assume the AOSP launcher (i.e. Nexus/Pixel launcher).  The steps to add the widget to the home screen may be slightly different based on the launcher used.  Other steps should be the same regardless of launcher.
1. Long-press home screen.
2. Tap ***Widgets*** option.
3. Scroll to ***Simplenote*** widget.
4. Long-press ***Note List (Light)*** widget.
5. Place ***Note List (Light)*** widget on home screen.
6. Notice note list shown in widget with light theme.
7. Long-press home screen.
8. Tap ***Widgets*** option.
9. Scroll to ***Simplenote*** widget.
10. Long-press ***Note List (Dark)*** widget.
11. Place ***Note List (Dark)*** widget on home screen.
12. Notice note list shown in widget with dark theme.

After adding the note list widgets to the home screen, follow the steps below for each style by choosing a different style in Step 5 each time.
1. Open navigation drawer.
2. Tap ***Settings*** item in drawer.
3. Tap ***Style*** item under ***Appearance*** section.
4. Notice ***Style*** screen with ***Default***, ***Matrix***, ***Publication***, ***Mono***, ***Sepia***, ***Black***, and ***Classic*** styles.
5. Tap ***Default***, ***Matrix***, ***Publication***, ***Mono***, ***Sepia***, ***Black***, or ***Classic*** style.
6. Notice selected style is shown in ***Styles***.
7. Tap back arrow in navigation bar.
8. Notice selected style is shown in ***Settings***.
9. Tap back arrow in navigation bar.
10. Notice selected style is shown in ***All Notes***.
11. Tap back arrow in navigation bar.
12. Notice selected style is shown in note list widgets.

### Review
Only one developer and one designer are required to review these changes, but anyone can perform the review.

#### Note
Contact me for an installable build.

### Release
These changes do not require release notes.  `RELEASE-NOTES.txt` will be updated once the styles feature is complete.
